subscription-service/src/test/kotlin/com/egm/stellio/subscription/utils/FixtureUtils.kt

```diff
@@ -11,6 +11,7 @@ fun gimmeRawSubscription(
     withQuery: Boolean = true,
     withGeoQuery: Boolean = true,
     withEndpointInfo: Boolean = true,
+    withNotifParams: Pair<Boolean, List<String>> = Pair(false, emptyList()),
```

Comment:
It would be clearer to pass a `Pair<NotificationFormat.FormatType, List<String>>` (hard to guess that `true` means that you want to `keyValues` format)

Reply:
OK